src/main/cms/cms_menu_misc.c

```diff
@@ -79,22 +79,36 @@ static const void *cmsx_menuRcConfirmBack(displayPort_t *pDisp, const OSD_Entry
     }
 }
 
+static int16_t rcDataInt[AUX4 + 1];
+
+static const void *cmsx_menuRcOnDisplayUpdate(displayPort_t *pDisp, const OSD_Entry *selected)
+{
+    UNUSED(pDisp);
+    UNUSED(selected);
+
+    for (int i = 0; i <= AUX4; i++) {
+        rcDataInt[i] = rcData[i];
```

Comment:
Should probably use `lround()` instead of truncation.

Reply:
Done!